tutorials/Python/setup.md

```diff
@@ -0,0 +1,157 @@
+# Setup instructions for the Oxford Computational Biochemistry Python course
+
+Included here are instructions to help you get started with the Oxford
+Computational Biochemistry Python course. The instructions are split into
+two parts:
+
+1. Setup
+
+These instructions should be followed prior to the tutorial. These will walk
+you through creating the appropriate conda environment for the tutorial and
+testing that everything works as expected.
+
+2. Starting the tutorial
+
+These instructions should be followed on the day of / when starting the tutorial.
+We assume here that you have completed the setup instructions on the same machine
+and that testing passed as expected.
+
+
+## 1. Setup
+
+Here we assume you have completed the main setup instructions and have done the
+following:
+
+1. You have installed an anaconda distribution
+2. You have access to a terminal
+3. You have downloaded the contents of this tutorial locally.
+
+
+### 1.1 Pre-install check
+
+Open a terminal and type the following:
```

Comment:
This also works on Windows? What about WSL?

Reply:
Yuup Anaconda prompt has full `conda` commands, it's magical. I dev-ed the Verify.ipynb straight from anaconda prompt.